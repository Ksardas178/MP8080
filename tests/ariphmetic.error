line 4: <ERROR> expected argument but recieved command
line 5: <ERROR> expected argument but recieved command
line 6: <ERROR> expected argument but recieved command
line 6: <ERROR> too few arguments in command. Expected 2
