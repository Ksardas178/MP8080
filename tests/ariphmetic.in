MOV 2+2 5
MOV B PSW
MOV d20 b11+h02
MOV d23 b11
MOV 2 5
MOV d23 b101
MOV C H
MOV 1 7
MOV B PSW
MOV d23 256
MOV 23 72
MOV 23-1 22+(11>>2)
MOV h2D d10*(b10-b10)
MOV 23 b111
MOV 23-1 2
MOV 23-1*2 3<<2
MOV (23-1)*2 2*2*2*3/4
MOV 2+2*2 7
MOV (2+2)*2/3 1
MOV (2+3)*2%3 2
MOV d23 d23+11
MOV o23-12 12+o7
MOV b101111 b0001+b1110
MOV b001001+b010010 1>>3
//MOV b001001 + b010010
MOV h2f b010+o1
MOV 2+2 4-1
MOV (68-4)*2/(2+2) 78
MOV (23-2)/2 9%2
MOV 2++ 1
MOV ++2 1
MOV --23 ++21
MOV 8<<2 --++2
MOV 8<<3>>3 3>>3