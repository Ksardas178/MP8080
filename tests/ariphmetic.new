initialized
initialized
line 2: <WARNING> argument adapted to match 3 8-bit value(s)
line 2: <ERROR> wrong register name recieved
line 3: <WARNING> argument adapted to match 3 8-bit value(s)
line 3: <ERROR> expected 3 argument(s)
line 2: get cmd with 3 args (2, 7, 729453341)
initialized
line 3: <WARNING> argument adapted to match 3 8-bit value(s)

[30;1mCode analysis results:
[0m______________________________
00000000: 00000011
00000001: 00000001
00000010: 00000010
00000011: 00000010
00000100: 00000111
00000101: 00000011
00000110: 00000001
00000111: 00000010
00001000: 00000000
00001001: 00000010
