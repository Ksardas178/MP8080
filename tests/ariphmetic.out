
[31;1mErrors: 3
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: 11001010
00000001: 00010111
00000010: 00101101
00000011: 11001010
00000100: 10000100
00000101: 00011101
